--- nemo-extensions-6.4.0/nemo-seahorse/tool/seahorse-pgp-operation.c.orig	2025-08-10 07:48:05.230791889 +0200
+++ nemo-extensions-6.4.0/nemo-seahorse/tool/seahorse-pgp-operation.c	2025-08-10 07:48:16.200856570 +0200
@@ -300,7 +300,9 @@ event_cb (void *data, gpgme_event_io_t t
         break;
 
     case GPGME_EVENT_NEXT_KEY:
+#if GPGME_VERSION_NUMBER < 0x020000
     case GPGME_EVENT_NEXT_TRUSTITEM:
+#endif
     default:
         /* Ignore unsupported event types */
         break;
